app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/LoginTest.java

```diff
@@ -0,0 +1,92 @@
+package yacs5e.ptpthingers.com.yacs5e_app;
+
+import android.content.Context;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.Button;
+import android.widget.TextView;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.concurrent.TimeUnit;
+
+import io.grpc.Grpc;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+
+public class LoginTest extends AppCompatActivity {
+
+    private Button mSendButton;
+    private TextView mResultText;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login_test);
+
+        mSendButton = (Button) findViewById(R.id.send_button);
+        mResultText = (TextView) findViewById(R.id.rpcText);
+    }
+
+    public void sendRPCRequest(android.view.View view) {
+        new GrpcTask().execute();
+    }
+
+    private class GrpcTask extends AsyncTask<Void, Void, String> {
+        private String mHost;
+        private int mPort;
+        private String username;
+        private String userpass;
+        private ManagedChannel mChannel;
+
+
+        @Override
+        protected void onPreExecute() {
+
+            //these vaues work only for test purposes
+            mHost = "18.216.25.170";
+            mPort = 13334;
+            username = "testUser";
+            userpass = "testPass";
+            mResultText.setText("");
+        }
+
+        @Override
+        protected String doInBackground(Void... nothing) {
```

Comment:
Check if eventual error is received and do we need to find another way to log them.